### PR TITLE
Add redis Eval

### PR DESCRIPTION
**Pull request by maximebedard**
Hi! We have scenarios where we need to test a set of redis scripts (using eval and evalsha), this PR adds support for eval and I'm currently working on adding evalsha too. 

Still very new to rust, let me know if there's anything I can improve here!

### CODE REVIEW

File: src/codec/redis_resp/gen.rs [hunk 1]
```diff
@@ -275,3 +292,20 @@ pub fn prepend(key: &str, value: &str) -> String {
         value
     )
 }
+
+pub fn eval(script: &str, keys: Vec<&str>) -> String {
```
Comment (brayniac) [inline src/codec/redis_resp/gen.rs]: Same here, add a docstring consistent with other commands

**Comment by maximebedard** [comment]
Going to close this PR as all the changes have been addressed in https://github.com/twitter/rpc-perf/pull/173

File: configs/eval.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# this example is testing redis eval and evalsha commands
```
Comment (kevyang) [inline configs/eval.toml]: only eval was added, correct? not evalsha?

Comment (maximebedard) [reply configs/eval.toml]: Correct, but I added both in #173 instead as I had to change a few things in this PR.

**Pull request closed without merging**